astroquery/gemini/core.py

```diff
@@ -341,7 +341,7 @@ def query_criteria(self, *rawqueryargs, coordinates=None, radius=0.3*units.deg,
                     raise ValueError("utc_date tuple should have two values")
                 if not isinstance(utc_date[0], date) or not isinstance(utc_date[1], date):
                     raise ValueError("utc_date tuple should have date values in it")
-                args.append("%s-%s" % utc_date[0].strftime("YYYYMMdd"), utc_date[1].strftime("YYYYMMdd"))
+                args.append("%s-%s" % (utc_date[0].strftime("YYYYMMdd"), utc_date[1].strftime("YYYYMMdd")))
```

Comment:
nit picking, but could change it to `.format()` from the old style formatting? had we insisted on it during the original review, this issue would have been smoked out.

Reply:
Done.